src/Service/LearningMaterialTextExtractor.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Entity\DTO\LearningMaterialDTO;
+use App\Exception\LearningMaterialTextExtractorException;
+use Exception;
+use Vaites\ApacheTika\Client;
+
+class LearningMaterialTextExtractor
+{
+    private bool $enabled = false;
```

Comment:
minor issue, but this could be a function 

```php
protected function isEnabled(): bool {
    return !empty($this-client);
}
```

Reply:
✅